website/src/docs/hotchocolate/v12/api-reference/apollo-federation.md

```diff
@@ -1,7 +1,295 @@
 ---
 title: Apollo Federation Subgraph Support
 ---
+> If you want to read more about Apollo Federation in general, you can head over to the [Apollo Federation documentation](https://www.apollographql.com/docs/federation/), which provides a robust overview and set of examples for this GraphQL architectural pattern. Many of the core principles and concepts are referenced within this document.
 
-> Note: Apollo Federation Support is coming with Hot Chocolate 12.6
+Hot Chocolate includes an implementation of the Apollo Federation v2 specification for creating Apollo Federated subgraphs. Through Apollo Federation, you can combine multiple GraphQL APIs into a single API for your consumers.
```

Comment:
I don't think it supports v2. I don't see the `@link` on my local HC federation schema with would make it a v1 schema:
![image](https://user-images.githubusercontent.com/45513122/219975000-d56f97b1-bf9b-4e47-a7fd-de48c55cd5b4.png)
@michaelstaib this is correct, right?

Reply:
Yeah, I think you are correct. Initially I thought it was 2.0, but as I wrote the docs and was looking at the spec more closely, I noticed the library is currently missing:

* The [`@shareable` attribute/directive](https://www.apollographql.com/docs/federation/federation-2/new-in-federation-2#shared-entity-fields)
* The ability to mark a [`@key` as `resolvable: false`](https://www.apollographql.com/docs/federation/federation-2/new-in-federation-2#changes-to-key)

So I'll update the docs to mention Apollo Federation 1.0 👍

Reply:
Resolved in b535586c6a2ceef2813c35f643643350d1c613cd

Reply:
Resolved in b535586c6a2ceef2813c35f643643350d1c613cd